src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/SnapshotResolverTest.cs

```diff
@@ -0,0 +1,274 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+#nullable disable
```

Comment:
Is this actually needed?

Reply:
This is just a philosophical thing for me. Tests with nullable disable makes sense because a NRE will fail the test. Having to have `AssumesNotNull` or similar pollutes it a bit without adding value

Reply:
`Assert.NotNull` is annotated so you can assert a failure.